lesson_08/homeworks.py

```diff
@@ -0,0 +1,34 @@
+
+def trees_total(apple, peer, plum):
+    '''Функція приймає кількість яблукб груш і слив, та виводить загальну кількусть дерев у саду'''
+    if apple >= 0 and peer >= 0 and plum >= 0:
+        return apple + peer + plum
+    else:
+        print("Задане значення не може бути негативним")

```

Comment:
```suggestion
        print("Задане значення не може бути негативним")
```
тут взагалі то не прінт а рейз повинен бути